app/src/components/BuyButton.jsx

```diff
@@ -0,0 +1,15 @@
+import shopSvg from '../assets/shop.svg';
+import { Link } from 'react-router-dom';
+
+
+export default function BuyButton() {
+
+    return <>
+        <Link className='buy-btn' to="/some-buy-page">
```

Comment:
classname? buy-button or button-buy, no need for hungarian notation in modern times